src/convert.ts

```diff
@@ -316,11 +316,11 @@ export default function convert(config: ConvertConfig): ESTreeNode | null {
      */
     if (
       additionalOptions.errorOnUnknownASTType &&
-      !AST_NODE_TYPES[customType]
+      !(customType in AST_NODE_TYPES)
```

Comment:
I don't think you need this, or the change on 323 if you update line 312 to this instead:

```ts
const customType = `TS${SyntaxKind[node.kind]}` as AST_NODE_TYPES;
```